JKNS-257: updates around new multi container sample pod templates in the openshift jenkins image (#39610)
openshift/openshift-docs: modules/images-other-jenkins-config-kubernetes.adoc

    </org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
----

> The following example also uses multiple containers. It uses {product-title} `ImageStream` image references to image streams that are running in the 'openshift' namespace:```

Reviewer: ```suggestion
The following example also uses multiple containers. It uses {product-title} `ImageStream` image references to image streams that are running in the `openshift` namespace:
```

    </org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
----

> Another example that uses multiple containers would be (where we use {product-title} ImageStream image references to

Reviewer: Please make this paragraph a single line.
```suggestion
The following example also uses multiple containers. It uses {product-title} `ImageStream` image references to refer to `ImageStreams` objects that are running in the 'openshift' namespace:```

> ImageStreams installed in the 'openshift' namespace that work when running in an {product-title} cluster):

Reviewer: ```suggestion
```
